schema/vehicle_types.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/GOFS-lite/GOFS-lite/blob/main/reference.md#vehicle_typesjson",
+  "description": "This file contains operating rules enabling on-demand services between zones or within the same zone, according to time windows and calendars. At least one operating rule MUST be defined. If start_pickup_window, end_pickup_window, and end_dropoff_window are not provided, it is assumed that the on-demand service is available at any hours of the day.",
+  "type": "object",
+  "properties": {
```

Comment:
This one is still missing a `version` field

Reply:
Thanks! Resolved in [f3a7f50](https://github.com/GOFS-lite/GOFS-lite/pull/18/commits/f3a7f509172b240ad7c91a1fdcc701244d2db0f4)